### DISCUSSION TITLE
v2.0.0: Licenses and Identifiers

### DISCUSSION THREAD

**Discussion by gcontini** [open-license-manager/licensecc#123] [Announcements]
This release addresses issuing licenses and identifier stability.
Compared to version 1.0 there is also a complete rearrangement of the project: the license generator is now a separate project. More tests, support for CentOS and lots of small fixes and improvements.

<hr /><em>This discussion was created from the release <a href='https://github.com/open-license-manager/licensecc/releases/tag/v2.0.0'>Licenses and Identifiers</a>.</em>

**Comment by GitHubUserC** [comment]
Could you please answer this issue?

https://github.com/open-license-manager/licensecc/issues/139

**Comment by antlassagne** [comment]
> Could you please answer this issue?
> d 
>   
> 
> https://github.com/open-license-manager/licensecc/issues/139

**Comment by CristinelStoica** [comment]
Is there a known issue with issuing licenses or license verification for Windows 11? It doesn't work for a client, and I can't figure out why. The license issuing and verification work for many different computers/users, but in this particular case it doesn't. Thank you!

**Reply by gcontini** [reply]
well the user may be doing something strange (like running in a VM or plugging some network card)
* Send lccinspector to him, it will output different pc identifiers.
* Ask him to run in different days, check which identifier is stable. (licensecc works with all the pc identifiers that are output by the inspector, no change)
* Check the output of lccinspector to see if you get some extra info on his execution environment.
* Ask the user what he is doing.

I have added issue #197 to make inspector output more useful. If you find the cause open an issue.

**Comment by ricardoyanez** [comment]
Is there an example of how to define the license path with LicenseLocation? I need to set it to a specific path and license name, e.g. /path/myproject.lic
